chef/cookbooks/cpe_screensaver/recipes/default.rb

```diff
@@ -50,10 +67,94 @@
                 {
                   'mcx_preference_settings' => {
                     'idleTime' => node['cpe_screensaver']['idleTime'],
-                    'askForPassword' =>
-                      node['cpe_screensaver']['askForPassword'],
-                    'askForPasswordDelay' =>
-                      node['cpe_screensaver']['askForPasswordDelay']
+                    'askForPassword' => node['cpe_screensaver']['askForPassword'],
+                    'askForPasswordDelay' => node['cpe_screensaver']['askForPasswordDelay']
+                  }
+                }
+              ]
+            }
+          }
+        },
+        {
```

Comment:
should these be added every time? What if i dont want to manage the screeen save type ?